### DISCUSSION TITLE
Error - You must provide an expression for the value to the right of the "-" operator.

### DISCUSSION THREAD

**Discussion by diakin** [ThioJoe/Windows-Super-God-Mode#4] [General]
You must provide an expression for the value to the right of the "-" operator.
C:\Users\Andrei\Downloads\Super_God_Mode.ps1:2400 pos:80
+                 $urlAssociations.PSObject.Properties | Where-Object { $_.Name - <<<< notin @("PSPath", "PSParentPath", "PSChildName", "PSDrive", "PSProvider") }
 | ForEach-Object {
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ExpectedValueExpression
--------------------------------------
Windows 7 x64

**Comment by ThioJoe** [comment]
Windows 7 uses a much older version of powershell so i'm not really surprised it doesn't work. I might be able to make some adjustments in the future to improve compatibility

**Reply by beedell-roke** [reply]
> https://github.com/ThioJoe/Windows-Super-God-Mode/discussions/4#discussioncomment-10440376

@ThioJoe, I'm not sure it's worth bothering. W7 has 3% marketshare, and it can't even install PS7 Core, per https://github.com/PowerShell/PowerShell/issues/14631#issuecomment-763978110.
